torch/csrc/jit/passes/quantization.cpp

```diff
@@ -575,51 +575,48 @@ void InsertQuantDeQuantImpl(
     }
   }
 
-  std::vector<Value*> values_to_quantize;
-  std::unordered_map<script::ModulePtr, script::Module>
-      child_modules_to_quantize;
   QuantizeHelper qh(module);
   std::stack<Block*> blocks_to_visit;
   blocks_to_visit.push(graph->block());
   while (!blocks_to_visit.empty()) {
     Block* b = blocks_to_visit.top();
     blocks_to_visit.pop();
-    for (Node* n : b->nodes()) {
+    for (auto it = b->nodes().begin(), end=b->nodes().end(); it != end;) {
```

Comment:
Nit: clang-format :-p

Reply:
can we make it a command hook or something? do you have the script?

Reply:
I use `git clang-format` - it formats not-committed changes. You could add it as a pre-commit hook, but I don't use it because it messes up reverts if the original version was ill-formatted.